### ISSUE TITLE
The code for test on blurry images from Levin and Lai dataset

### ISSUE THREAD

**Issue by Alvacv** [xl-tang3/UAUDeblur#4]
I tried to test the dataset of Levin and Lai by modifying the input of demo.py code, but it didn't reach the recovery quality of Table1. Can u publish the code of the corresponding dataset?

**Comment by xl-tang3** [comment]
Thanks for your attention. It might be helpful to set the value of the parameter 'load_pretrained' to 0 when testing on new images. By the way, the results would be more favorable if you go on with save and load tricks for the optimization.

**Comment by Alvacv** [comment]
Thanks for your reply. I have tried not to use the pretrained model, but the quantityresults are still somewhat different from table1. I want to ask about the code and  corresponding training parameters for these two dataset.

**Comment by xl-tang3** [comment]
Table1 lists the average results for the images of different types of kernels. Test on a single image definitely produces different results.

But I would recommend to initialize weights for the residual network by capturing the raw residual and tune with lower learning rate on this basis. This is what we do to attain the most pleasing results.

**Comment by Alvacv** [comment]
How to pretrain a residual network whose input is noise？

**Comment by xl-tang3** [comment]
> 

Just do weight initialization, which is included in the code.

**Comment by yangxinwang** [comment]
Did your experiment succeed? @Alvacv
